File: 73-set-matrix-zeroes/set-matrix-zeroes.cpp
class Solution {
public:
    void setZeroes(vector<vector<int>>& matrix) {
        vector<vector<int>>posi;
        for(int i = 0;i<matrix.size();i++){
            for(int j =0;j<matrix[i].size();j++){
                if(matrix[i][j]==0){
                    posi.push_back({i,j});
                }
            }

        }
        for(auto k:posi){
            for(int a =0;a<matrix[k[0]].size();a++){
                matrix[k[0]][a] = 0;
            }
            for(int b = 0; b<matrix.size();b++){
                matrix[b][k[1]] = 0;
            }
        }

    }
};

File: 73-set-matrix-zeroes/set-matrix-zeroes.py
class Solution:
    def setZeroes(self, matrix: List[List[int]]) -> None:
        """
        Do not return anything, modify matrix in-place instead.
        """
        rl=len(matrix)
        cl=len(matrix[0])
        row=[0]*rl
        col=[0]*cl
        for i in range(0,rl):
            for j in range(0,cl):
                if(matrix[i][j]==0):
                    row[i]=1
                    col[j]=1
        for n in range(0,rl):
            for m in range(0,cl):
                if (row[n]==1 or col[m]==1):
                    matrix[n][m]=0
                    